app/assets/json/translations/locale-en.json

```diff
@@ -560,7 +563,9 @@
             "PUBLIC_BENEFITS": "Public benefits record",
             "SCHOOL_RECORD": "School record",
             "TELEPHONE_BILL": "Telephone bill (land line only)",
-            "WATER_BILL": "Water bill"
+            "WATER_BILL": "Water bill",
+            "CA_LICENSE": "CA ID or Driver's License",
```

Comment:
@slowbot the CA state official term for this is "Driver License" not "Driver's License" - check out CA DMV: https://www.dmv.ca.gov/portal/dmv/dmv/dl. would it be ok for us to change "CA ID or Driver's License" to "CA ID or Driver License" in the label here? @creativecoder would that be too confusing since the Salesforce picklist value does use the possessive?

Reply:
We can update and I'd recommend changing the Salesforce picklist values as well to keep things consistent.

Reply:
ok, if @slowbot is ok with it then i can ask Vertiba to update that picklist value